app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -84,13 +84,13 @@ private fun LoadedContent(parties: List<Party>, openDetails: (Int) -> Unit) {
 @Composable
 private fun PartyTile(
 	party: Party,
-	openDetails: (Int) -> Unit,
+	onPartyClick: (Int) -> Unit,
```

Comment:
Here, I would suggest to simplify it to standalone `onClick`

Reply:
Updated. Thank you.

Reply:
You're welcome.